### PR TITLE
Pass GLI options and args to hook

### PR DESCRIPTION
**Pull request by matl33t**

### CODE REVIEW

**Comment by apurvis** [comment]
did you mean to assign this to me?

**Comment by matl33t** [comment]
Sure, but probably not much to comment on here. Got any ideas on how this can be better?

File: lib/broadside/gli/global.rb [hunk 1]
```diff
@@ -49,7 +52,7 @@ def call_hook(type, command)
     Broadside.config.logger.level = ::Logger.const_get(global[:loglevel].upcase)
   end
 
-  call_hook(:prehook, command)
+  call_hook(:prehook, command, options, args)
```
Comment (apurvis) [inline lib/broadside/gli/global.rb]: don't you need to do this for `:posthook` too?

**Comment by apurvis** [comment]
(also make sure you update the wiki)

**Comment by apurvis** [comment]
👍

**Pull request merged**